digitaltwin_client/samples/readme.md

```diff
@@ -4,24 +4,79 @@ This directory contains samples demonstrating creating and using Digital Twin in
 
 ## Directory contents
 
-* [digitaltwin_sample_device](./digitaltwin_sample_device) - executable that interacts with the other Digital Twin interfaces in this directory.
-* [digitaltwin_sample_ll_device](./digitaltwin_sample_ll_device) - executable that interacts with the other Digital Twin interfaces in this directory, using the \_LL\_ layer and enabling Device Provisioning Service (DPS).
-* [digitaltwin_sample_device_info](./digitaltwin_sample_device_info) - library that implements a sample device info Digital Twin interface.  This interface reports information about the device - such as OS version, amount of storage, , etc.
-* [digitaltwin_sample_environmental_sensor](./digitaltwin_sample_environmental_sensor) library that implements a sample environmental sensor Digital Twin interface.  This interface demonstrates all concepts of the implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
+### Executable Source Code
+* [digitaltwin\_sample\_device](./digitaltwin_sample_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation when using Iot Hub.
```

Comment:
Maybe @jspaith can confirm but I think the "T" in IoT is [capitalized](https://azure.microsoft.com/en-us/services/iot-hub/)
```suggestion
* [digitaltwin\_sample\_device](./digitaltwin_sample_device) - This executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation when using IoT Hub.
```